sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/src/Config/BlobsExtensionConfigProvider.cs

```diff
@@ -205,6 +210,35 @@ public async Task<HttpResponseMessage> ConvertAsync(HttpRequestMessage input, Ca
             return (T)blob.BlobClient;
         }
 
+        private static ParameterBindingData ConvertToParameterBindingData(BlobAttribute blobAttribute)
+        {
+            var blobPath = BlobPath.ParseAndValidate(blobAttribute.BlobPath);
+
+            dynamic blobDetails = new ExpandoObject();
+            blobDetails.Connection = blobAttribute.Connection ?? Constants.DefaultAzureStorageConnectionName;
+            blobDetails.BlobName = blobPath.BlobName;
+            blobDetails.ContainerName = blobPath.ContainerName;
+
+            return CreateParameterBindingData(blobDetails);
+        }
+
+        private static ParameterBindingData ConvertToParameterBindingData(BlobBaseClient input, BlobTriggerAttribute blobAttribute)
+        {
+            dynamic blobDetails = new ExpandoObject();
+            blobDetails.Connection = blobAttribute.Connection ?? Constants.DefaultAzureStorageConnectionName;
+            blobDetails.BlobName = input.Name;
+            blobDetails.ContainerName = input.BlobContainerName;
+
+            return CreateParameterBindingData(blobDetails);
+        }
+
+        private static ParameterBindingData CreateParameterBindingData(object blobDetails)
+        {
+            var blobDetailsBinaryData = new BinaryData(blobDetails);
+            var bindingData = new ParameterBindingData("1.0", "AzureStorageBlobs", blobDetailsBinaryData, "application/json");
```

Comment:
Recommend defining a constant for the extension name string https://github.com/Azure/azure-sdk-for-net/blob/dc39a5e449830403a3fddcb46d2ab6d95e3942c0/sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/src/Config/BlobsExtensionConfigProvider.cs#L30 and use in both places